js/build.js

```diff
@@ -206,41 +204,44 @@ Fliplet.Widget.instance('chat', function (data) {
       'transition': 'all ' + animationSpeed + 'ms ease-out'
     });
 
-    if (e.velocityX > PAN_VELOCITY_BOUNDARY) {
+    if (distanceX > distanceY && velocityX > PAN_VELOCITY_BOUNDARY) {
       closeConversation();
       return;
     }
 
-    if (e.deltaX < screenWidth / PAN_WINDOW_FRACTION || distanceX > distanceY) {
-      $chatOverlay.css({
-        '-webkit-transform': 'translate3d(0, 0, 0)',
-        'transform': 'translate3d(0, 0, 0)'
-      });
-      $list.css({
-        '-webkit-transform': 'translate3d(-25%, 0, 0)',
-        'transform': 'translate3d(-25%, 0, 0)'
-      });
-    } else {
+    if (deltaX > screenWidth / PAN_WINDOW_FRACTION && distanceX > distanceY) {
       closeConversation();
+      return;
     }
-  }
-
-  function openConversation(conversationId) {
-    setTimeout(function() {
-      $('[data-conversation-id="'+ conversationId +'"]').addClass('open');
-    }, 1);
 
     $chatOverlay.css({
       '-webkit-transform': 'translate3d(0, 0, 0)',
```

Comment:
jQuery `.css()` does at vendor prefixes, so you can skip these to simplify the code.

See https://stackoverflow.com/questions/17487716/does-css-automatically-add-vendor-prefixes

Reply:
Be sure to search for all the vendor prefixes